packages/manager/apps/pci-block-storage/src/pages/new/form.type.ts

```diff
@@ -1,9 +1,11 @@
-import { TAddon } from '@ovh-ux/manager-pci-common';
-import { TLocalisation } from '@/api/hooks/useRegions';
+import { TVolumeAddon, TVolumePricing } from '@/api/data/catalog';
+import { TRegion } from '@/api/data/regions';
```

Comment:
As a global improvement (in a future refactoring), we should not make the form depending on DTO but rather than front-end entities to prevent the µapp to be coupled to API

Reply:
Yes, this will be removed when we remove the stepper and use the new region selector